apis/filesystem/src/test/java/org/jclouds/filesystem/integration/FilesystemBlobIntegrationTest.java

```diff
@@ -49,27 +51,24 @@ protected Properties setupProperties() {
    // https://bugs.openjdk.java.net/browse/JDK-8030048
    @Override
    public void checkContentMetadata(Blob blob) {
-      if (!isMacOSX()) {
-         super.checkContentMetadata(blob);
-      }
+      checkExtendedAttributesSupport();
+      super.checkContentMetadata(blob);
    }
 
    // Mac OS X HFS+ does not support UserDefinedFileAttributeView:
    // https://bugs.openjdk.java.net/browse/JDK-8030048
    @Override
    protected void checkContentDisposition(Blob blob, String contentDisposition) {
-      if (!isMacOSX()) {
-         super.checkContentDisposition(blob, contentDisposition);
-      }
+      checkExtendedAttributesSupport();
+      super.checkContentDisposition(blob, contentDisposition);
    }
 
    // Mac OS X HFS+ does not support UserDefinedFileAttributeView:
    // https://bugs.openjdk.java.net/browse/JDK-8030048
    @Override
    protected void validateMetadata(BlobMetadata metadata) throws IOException {
-      if (!isMacOSX()) {
-         super.validateMetadata(metadata);
-      }
+      checkExtendedAttributesSupport();
```

Comment:
This is what I mentioned in my previous review -- this will cause a test which checks many metadata, including non-xattr, to skip.  We should only skip full tests.